### PR TITLE
HON-93 Add options to the create_or_update_site command to populate SiteConfiguration fields

### PR DESCRIPTION
**Pull request by douglashall**
@vkaracic @mjfrey 

Associated edx-documentation PR is [here](https://github.com/edx/edx-documentation/pull/1258).

### CODE REVIEW

File: ecommerce/core/management/commands/create_or_update_site.py [hunk 1]
```diff
@@ -94,6 +94,18 @@ def add_arguments(self, parser):
...
+                            action='store',
+                            dest='payment_support_email',
+                            type=str,
+                            required=False,
+                            help='Email address displayed to user for payment support')
+        parser.add_argument('--payment-support-url',
+                            action='store',
+                            dest='payment_support_url',
+                            type=str,
+                            required=False,
```
Comment (vkaracic) [inline ecommerce/core/management/commands/create_or_update_site.py:107]: I think this is required. A site configuration without this value gives out errors everywhere (that it's missing).

Comment (douglashall) [reply ecommerce/core/management/commands/create_or_update_site.py:107]: It is not required. The model allows for this field to be [blank](https://github.com/edx/ecommerce/blob/master/ecommerce/core/models.py#L94).

This field is only used in the basket [template](https://github.com/edx/ecommerce/blob/282ebf64b0c7b1ed0cd73d7a0f34936bd548c88a/ecommerce/templates/oscar/basket/partials/basket_content.html#L204) when the basket is empty.

**Comment by douglashall** [comment]
@vkaracic Tests now passing on this one.

File: ecommerce/core/management/commands/create_or_update_site.py (hunk 1 above)
Comment (vkaracic) [reply ecommerce/core/management/commands/create_or_update_site.py:107]: You are correct. With a new database I don't get any errors, I'm not sure why I kept getting them on my old database.

**Review by vkaracic — APPROVED** [review]
LGTM
Thanks for adding this!

**Pull request merged**